src/protocols/ssh/ssh.c

```diff
@@ -323,6 +323,9 @@ void* ssh_client_thread(void* data) {
         /* Track total amount of data read */
         int total_read = 0;
 
+        /* Timer for polling socket activity */
+        int timer;
```

Comment:
Missed a few spots for 8c24c77's changes ("GUACAMOLE-203: Change timer to timeout").